src/ArcxAnalyticsSdk.ts

```diff
@@ -59,6 +59,19 @@ export class ArcxAnalyticsSdk {
 
     this._registerSocketListeners(socket)
 
+    socket.once('error', (error) => {
+      if (error.name === 'InternalServerError') {
```

Comment:
Also, realized that in prod it's not an `InternalServerError` but rather a `BadRequestError`. See here: 

![image](https://user-images.githubusercontent.com/5834876/225098304-8a7d0467-78cf-477a-8f10-cc3a951a739e.png)

Identity ID: `32198a83d76a6993e501f643`